gatsby-source-graphcms/README.md

```diff
@@ -22,10 +22,18 @@
     <a href="https://gatsby-source-graphcms.vercel.app">Demo</a> • <a href="https://github.com/GraphCMS/gatsby-starter-graphcms-blog">gatsby-starter-graphcms-blog</a> • <a href="https://slack.graphcms.com">Join us on Slack</a>  • <a href="https://app.graphcms.com">Login to GraphCMS</a> • <a href="https://twitter.com/GraphCMS">@GraphCMS</a>
 </p>
 
+<br />
```

Comment:
Is this needed?

Reply:
It gives a little margin from the badges. It looks better this way IMO.